modules/candidate_parameters/jsx/ProbandInfo.js

```diff
@@ -97,6 +97,66 @@ var ProbandInfo = React.createClass(
         dob2Required = true;
       }
 
+      var extraParameterFields = [];
+      var extraParameters = this.state.Data.extra_parameters;
+      for (var key2 in extraParameters) {
+        if (extraParameters.hasOwnProperty(key2)) {
+          var paramTypeID = extraParameters[key2].ParameterTypeID;
+          var name = 'PTID' + paramTypeID;
+          var value = this.state.Data.parameter_values[paramTypeID];
+
+          switch (extraParameters[key2].Type.substring(0, 3)) {
```

Comment:
I'm not really sure what `extraParameters[key2].Type.substring(0, 3)` is.
I can infer it from the `case` statements but IMO it would be easier to read/understand if this were a named variable:

```
var extraParamType = extraParameters[key2].Type.substring(0, 3)
switch (extraParamType) { ... }
```
(I'm just guessing that it is the extra parameter type)